backend/src/v5/processors/teamspaces/projects/models/commons/assets/bundles.js

```diff
@@ -0,0 +1,30 @@
+/**
+ *  Copyright (C) 2025 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const { getAssetList } = require('../../../../../../models/bundles');
+
+const JsonAssets = { };
+
+JsonAssets.getRepoBundleInfo = async (teamspace, model, revision, subModels) => {
```

Comment:
I would just keep a containerList as a param and edit container processor method to pass it an array cause I find both model and subModels params a bit weird